internal/olm/manager.go

```diff
@@ -40,17 +40,21 @@ type Manager struct {
 }
 
 func (m *Manager) initialize() (err error) {
+	errChan := make(chan error, 1)
+	defer close(errChan)
 	m.once.Do(func() {
 		if m.Client == nil {
 			cfg, err := config.GetConfig()
 			if err != nil {
 				err = errors.Wrapf(err, "failed to get Kubernetes config")
```

Comment:
Directly pass `err` to `errChan` here and below.

Reply:
Do.